meson.build

```diff
@@ -935,9 +935,11 @@ if features['direct3d']
 endif
 
 drm = dependency('libdrm', version: '>= 2.4.105', required: get_option('drm'))
-features += {'drm': drm.found() and (features['vt.h'] or features['consio.h'] or features['wsdisplay-usl-io.h'])}
+libdisplay_info = dependency('libdisplay-info', required: get_option('drm'))
```

Comment:
Would it be useful to parse EDID and use that info also in non-drm scenarios?

Reply:
When would edid data be useful for us if we aren't the ones modesetting?

Reply:
I can't think of one. On the wayland side, it's actually possible to get most of this information via `xx-color-management-v4` anyways.